docs/embedding/sdk/dashboards.md

```diff
@@ -101,35 +101,29 @@ See [customizing interactive questions](./questions.md#customizing-interactive-q
 
 ## Dashboard plugins
 
-### `dashcardMenu`
+### `dashboardCardMenu`
 
 This plugin allows you to add, remove, and modify the custom actions on the overflow menu of dashboard cards. The plugin appears as a dropdown menu on the top right corner of the card.
 
 The plugin's default configuration looks like this:
 
 ```typescript
 const plugins = {
-  dashboard: {
-    dashcardMenu: {
-      withDownloads: true,
-      withEditLink: true,
-      customItems: [],
-    },
+  dashboardCardMenu: {
```

Comment:
Was a decision made on this? The idea when I made this thing was that we might have more `dashboard` properties in the future, so if someone wanted to make a dashboard config then it would all be nested under `dashboard`

Reply:
@oisincoveney this is based on [this comment](https://www.notion.so/Tech-Review-SDK-s-API-surface-and-propose-adjustments-13e69354c90180a7974fe78c1afce3a1?d=14a69354c90180bbb045001cd4565706&pvs=4#13e69354c90180f5b538c52d5d68ffd3) because we used to use the `SdkDashCardMenuPluginsConfig` type for the `dashboard` key which doesn't match:

```tsx
dashboard?: SdkDashCardMenuPluginsConfig;
```

We could go the other way by renaming `SdkDashCardMenuPluginsConfig` to `MetabaseDashboardPluginsConfig` and renaming `dashboard.dashcardMenu` to `dashboard.dashboardCardMenu`

Reply:
awesome! now we just need to make sure we add more options to the dashboard to make the change worthwhile 😄